packages/client/src/components/create-client/create-client.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import {
+  ButtonBar,
+  K8sResourceKind,
+  NamespaceModel,
+  ResourceDropdown,
+  getName,
+  useCustomTranslation,
+} from '@odf/shared';
+import { k8sCreate } from '@openshift-console/dynamic-plugin-sdk';
+import * as _ from 'lodash-es';
+import { useNavigate } from 'react-router-dom-v5-compat';
+import {
+  Form,
+  Button,
+  FormGroup,
+  TextInput,
+  TextArea,
+  TextContent,
+  Text,
+  TextVariants,
+  ActionGroup,
+  ValidatedOptions,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+} from '@patternfly/react-core';
+import { StorageClientModel } from '../../models';
+import './create-client.scss';
+import '../../../style.scss';
+
+const CreateStorageClientHeader: React.FC = () => {
+  const { t } = useCustomTranslation();
+  return (
+    <div className="odf-create-storage-client__header">
+      <TextContent>
+        <Text component={TextVariants.h1}>{t('Create StorageClient')}</Text>
+        <Text component={TextVariants.small}>
+          {t('Create a Storageclient to connect to a Data Foundation system.')}
+        </Text>
+      </TextContent>
+    </div>
+  );
+};
+
+const isValidURL = (userInput: string) => {
+  if (!userInput) {
+    return ValidatedOptions.default;
+  }
+  try {
+    (() => new URL(userInput))();
+    const port = userInput.split(':').pop();
+    if (port !== '' && _.isInteger(Number(port))) {
+      return ValidatedOptions.default;
+    } else {
+      return ValidatedOptions.error;
+    }
+  } catch {
+    return ValidatedOptions.error;
+  }
+};
+
+const CreateStorageClient: React.FC = () => {
+  const [name, setName] = React.useState('');
+  const [namespace, setNamespace] = React.useState('');
+  const [address, setAddress] = React.useState('');
+  const [ticket, setTicket] = React.useState('');
+  const [inProgress, setProgress] = React.useState(false);
+  const [error, setError] = React.useState<Error>();
+
+  const { t } = useCustomTranslation();
+  const navigate = useNavigate();
+
+  const submit = () => {
+    setProgress(true);
+    k8sCreate({
+      model: StorageClientModel,
+      data: {
+        metadata: {
+          name,
+          namespace,
+        },
+        spec: {
+          storageProviderEndpoint: address,
+          onboardingTicket: ticket,
+        },
+      },
+    })
+      .then(() => {
+        setProgress(false);
+      })
+      .catch((err) => {
+        setProgress(false);
+        setError(err);
+      });
+  };
+
+  return (
+    <div>
```

Comment:
is `div` intentional ?
```suggestion
    <>
```

Reply:
yes